subiquity/controllers/installpath.py

```diff
@@ -58,40 +60,15 @@ def choose_path(self, path):
         self.model.path = path
         getattr(self, 'install_' + path)()
 
+    def install_execshell(self):
+        if not self.opts.dry_run:
+            utils.disable_subiquity()
+        self.signal.emit_signal('quit')
```

Comment:
??  This looks like it exits subiquity.  Surely we expect 'drop to shell' to be a subshell like in d-i, that when you exit takes you back to subiquity.  Or is this doing something else that's not obvious from the code?

Reply:
utils.disable_subiquity() will never retrun, as it stops the unit and starts the getty units. Please see https://github.com/CanonicalLtd/subiquity/blob/f9b9414de1b558daceb2bf9e46552b765f0e8fdc/subiquitycore/utils.py#L170

The quit, is for the dry run mode..... maybe i should invert the code. This is copy & paste from the installprogress crash handler, which already had the 'Exit to shell' button.